dtmi/azureiot/phone_as_a_device/sensors/location-1.json

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:azureiot:Phone_As_A_Device:Sensors:Location;1",
+    "@type": "Interface",
+    "displayName": {
+      "en": "Location"
+    },
+    "contents": [
+      {
+        "@type": "Property",
+        "displayName": {
+          "en": "Enabled"
+        },
+        "name": "enabled",
+        "schema": "boolean",
+        "writable": true
+      },
+      {
+        "@type": "Telemetry",
+        "displayName": {
+          "en": "Location"
+        },
+        "name": "geolocation",
+        "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Location:position;1"
```

Comment:
Avoid custom schemas for types that are already provided.

you can use `point` (defined by central only), or Location with geopoint, eg:

```json
      {
        "@type": ["Telemetry", "Location"] ,
        "name": "location",
        "schema": "geopoint"
      }  
```